packages/manager/apps/restricted/src/hooks/useMinimalRights/useMinimalRights.ts

```diff
@@ -0,0 +1,11 @@
+import { useContext } from 'react';
+import Context from '@/context';
+import { MINIMUM_RIGHTS } from './constants';
+
+export default function useMinimalRights() {
+  const { ovhSubsidiary } = useContext(Context);
+  return MINIMUM_RIGHTS.filter(
+    ({ subsidiariesExcluded }) =>
+      !subsidiariesExcluded || !subsidiariesExcluded.includes(ovhSubsidiary),
```

Comment:
```suggestion
      !subsidiariesExcluded?.includes(ovhSubsidiary),
```

Sonar tips : https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=14845&id=ovh_manager&open=AZRKx-EhVoOmZhQzYt6O